pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -210,7 +225,7 @@ func RunListBuildWebHooks(f *clientcmd.Factory, out, errOut io.Writer, name stri
 		return err
 	}
 
-	if isBuild {
+	if resource == "builds" {
```

Comment:
switch on resource and handle buildconfigs, builds, and default error. In general, I want to make sure we don't make assumptions about resources for when we want to accept other resources